lib/Dialect/AIE/IR/AIEDeviceModel.cpp

```diff
@@ -0,0 +1,207 @@
+//===- AIEDialect.cpp -------------------------------------------*- C++ -*-===//
+//
+// This file is licensed under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// (c) Copyright 2019 Xilinx Inc.
+//
+//===----------------------------------------------------------------------===//
+
+#include "aie/Dialect/AIE/IR/AIEDeviceModel.h"
+#include "llvm/ADT/DenseSet.h"
+#include "llvm/ADT/SmallSet.h"
+
+using namespace llvm;
+
+namespace xilinx {
+namespace AIE {
```

Comment:
[Style guide nit](https://llvm.org/docs/CodingStandards.html#use-namespace-qualifiers-to-implement-previously-declared-functions). I only mention because I've been bitten by the bug in the link and much of our code follows the pattern of opening namespace blocks.